index.js

```diff
@@ -3,73 +3,98 @@ $("#selectOption").val(localStorage.getItem("optionValue"));
 
 //Mapping the data from the server by:
 const mapping = {
-    name: { fn: _getName, isVisible: true },
-    city: { fn: _getCity, isVisible: true },
-    address: { fn: _getAddress, isVisible: true },
-    imgSrc: { fn: _getImage, isVisible: true },
+  name: { fn: _getName, isVisible: true },
+  city: { fn: _getCity, isVisible: true },
+  address: { fn: _getAddress, isVisible: true },
+  imgSrc: { fn: _getImage, isVisible: true },
 };
 
-function _getName (user) {
-    return `${user.name.first} ${user.name.last}`;
-};
+function _getName(user) {
+  return `${user.name.first} ${user.name.last}`;
+}
 
-function _getAddress (user) {
-    return `${user.location.street.name} ${user.location.street.number}`;
-};
+function _getAddress(user) {
+  return `${user.location.street.name} ${user.location.street.number}`;
+}
 
-function _getCity (user) {
-    return `${user.location.city}`;
-};
+function _getCity(user) {
+  return `${user.location.city}`;
+}
 
-function _getImage (user) {
-    return `${user.picture.medium}`;
-};
+function _getImage(user) {
+  return `${user.picture.large}`;
+}
 
-async function init () {
-    try {
-        const selectValue = $("#selectOption").val();
-        const response = await getUsersFromServer({ url: `https://randomuser.me/api/?results=${selectValue}` });
-        const { results } = response;
-        draw(results)
-    } catch (err) {
-        console.log(err);
-        alert(`message: ${err.statusText} , status: ${err.status}`);
-    };
-};
+const cardsContainer = $("#cardsContainer");
 
-function draw(arrOfObjects) {
+async function init() {
+  try {
+    const selectValue = $("#selectOption").val();
+    const response = await getUsersFromServer({
+      url: `https://randomuser.me/api/?results=${selectValue}`,
+    });
+    const { results } = response;
+    draw(results);
+  } catch (err) {
+    console.log(err);
+    alert(`message: ${err.statusText} , status: ${err.status}`);
+  }
+}
 
-    const mappedUsers = arrOfObjects.map((user)=>{
-        return getMappedUser(user);
+function draw(arrOfObjects) {
+  const mappedUsers = arrOfObjects.map((user) => {
+    return getMappedUser(user);
+  });
+  
+//   //DRAW
+    cardsContainer.empty();
+    const cards = mappedUsers.map((user) => {
+        return drawCards(user)
     })
-    console.log(mappedUsers);
+}
 
-    //DRAW
-};
+function drawCards(user) {
+    const mainDiv = $(`<div></div>`).addClass("card").css({width: "18rem", display: "inline-block"});
```

Comment:
adding class and style ?
maybe put another class and insert the relevant style into